Core/Config.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2012- PPSSPP Project.
+﻿// Copyright (c) 2012- PPSSPP Project.
```

Comment:
Our `.editorconfig` specifies no byte order mark in this file.
https://github.com/hrydgard/ppsspp/blob/master/.editorconfig#L5

If you think we should be using byte order marks in utf-8 (I disagree), can you change that file?  Otherwise, I think this should be saved without such a mark.

You can install this extension to automatically ensure your Visual Studio is configured per the `.editorconfig` definitions:
https://marketplace.visualstudio.com/items?itemName=EditorConfigTeam.EditorConfig

Just set it and forget it.

-[Unknown]